apps/galaxy/src/utils/tokenService.ts

```diff
@@ -0,0 +1,32 @@
+export const setToken = (token: string) => {
+  localStorage.setItem("accessToken", token);
+};
+
+export const getToken = () => {
+  if (typeof window !== "undefined") {
+    return localStorage.getItem("accessToken");
+  }
+};
+
+export const checkTokenExpiration = () => {
+  const token = getToken();
+  const utcTimestamp = new Date();
+  return utcTimestamp > parseJwtToken(token as string);
+};
+
+const parseJwtToken = (token: string) => {
```

Comment:
actually suggest to use a lib to parse Jwt token

Reply:
I searched some Jwt token libraries and the most popular one is 29.7kb, so I thought I can use 10 lines of code to parse it instead of importing a library. But, that may be overthinking, it is true we can use a library directly to make a more readable and simple code. So I changed to using the **Jwt-decode** library rn.